src/audio-worklet/migration/spn-recorder/app.js

```diff
@@ -121,7 +124,7 @@ function setupScriptProcessor(recordingProperties, passSampleToVisualizers) {
       
       if (recordingLength > recordingProperties.maxFrameCount) {
         isRecording = !isRecording;
-
+        window.alert("The recording length reach the max limit!");
```

Comment:
I advise against using `alert`. Probably stopping and displaying some warning texts in `div` would be better.